doc/release-process.md

```diff
@@ -191,14 +180,44 @@ Commit your signature to gitian.sigs:
 
 ### After 3 or more people have gitian-built and their results match:
 
-- Upload gitian zips to SourceForge
+- Perform code-signing.
 
-- Announce the release:
+    - Code-sign Windows -setup.exe (in a Windows virtual machine using signtool)
+
+    - Code-sign MacOSX .dmg
+
+  Note: only Max has the code-signing keys currently.
 
   - Add the release to dogecoin.com
 
+- Create `SHA256SUMS.asc` for builds, and PGP-sign it. This is done manually.
+  Include all the files to be uploaded. The file has `sha256sum` format with a
+  simple header at the top:
+
+```
+Hash: SHA256
+
+0060f7d38b98113ab912d4c184000291d7f026eaf77ca5830deec15059678f54  dogecoin-x.y.z-linux.tar.gz
+...
+```
+
+- Upload zips and installers, as well as `SHA256SUMS.asc` from last step, to the dogecoin.com Github repo
+
+- Update dogecoin.com version
+
+  - Make a pull request to add a file named `YYYY-MM-DD-vX.Y.Z.md` with the release notes
+  to https://github.com/bitcoin/bitcoin.org/tree/master/_releases
```

Comment:
I guess we should _not_ do this :D  

Actual process is: I pull the repo, commit and push it again ;)